src/tests/Missions.test.js

```diff
@@ -0,0 +1,18 @@
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Missions from '../component/Misssions';
+import store from '../Redux/store';
+
+jest.setTimeout(10000);
+describe('Missions', () => {
+  it('It has to match the snapshot', () => {
+    const Mission = render(
+      <>
+        <Provider store={store}>
+          <Missions />
+        </Provider>
+      </>,
+    );
+    expect(Mission).toMatchSnapshot();
+  });
+});
```

Comment:
[Optional]

- [ ] As pointed out by the previous reviewer, you still have failing tests. Kindly fix the issues 👍 

![image](https://user-images.githubusercontent.com/50721479/204003924-21c7a9b5-8c51-42cd-ad5d-27cf42f1b3a3.png)